scapy/layers/dns.py

```diff
@@ -943,7 +946,7 @@ class DNSRR(Packet):
                                         length_from=lambda pkt: pkt.rdlen),
                                lambda pkt: pkt.type in [2, 3, 4, 5, 12]),
                            # TEXT
-                           (DNSTextField("rdata", [],
+                           (DNSTextField("rdata", [""],
```

Comment:
I also don't really like this default. This is very un-Scapy. What other options would we have to fix this issue?

Reply:
I don't feel strongly about this so I can just remove this but it would mean that technically TXT RRs violate the RFC by default.

Reply:
That being said I agree that the default should be changed in this PR. It's a separate issue and should probably be discussed in a separate PR. I removed that part.

Reply:
Please allow me to change my mind. Your proposed default makes more sense. ^^'

Reply:
I brought it back